spec/models/remote_rental_spec.rb

```diff
@@ -11,6 +11,28 @@
   it { is_expected.to validate_uniqueness_of(:uid).allow_nil }
   it { is_expected.to validate_presence_of(:remote_account) }
 
+  describe '.connected' do
+    subject { described_class.connected }
+    let(:connection_1) { create(:connection) }
+    let(:remote_rental_1) { connection_1.remote_rental }
+    let(:connection_2) { create(:canceled_connection) }
+    let(:remote_rental_2) { connection_2.remote_rental }
+    let(:remote_rental_3) { create(:remote_rental, rental: nil) }
+
+    it { is_expected.to eq [remote_rental_1] }
```

Comment:
I think it would be better to add description here that it returns connected remote rentals, with not canceled connection or sth like that. Much easier to immediately see what's going on here. Also doesn't matter much in this case, but in general it's better to use `match_array` instead of `eq` since I think it doesn't care about order of returned results

Might be good idea to name objects more expressive as well - `not_connected_remote_rental` `connected_remote_rental` etc.